### PR TITLE
Add method team get_by_id()

### LINKED ISSUE #46: Get specific team details
https://app.swaggerhub.com/apis/Veracode/veracode-identity_api/1.0#/team/getTeamUsingGET

This is to get the list of members in a team, which is required if we want to easily add/remove people from teams.

Currently:

### Adding a user to a team

```
            VeracodeAPI().update_user(
                user["user_id"],
                {
                    "teams": [
                        {
                            "team_id": team_to_add_id,
                            "team_legacy_id": team_to_add_legacy_id,
                        }
                    ]
                },
            )
```

### Remove a user from a team

You can't, that's because the above API is [incremental only](https://github.com/veracode/veracode-api-py/blob/main/veracode_api_py/identity.py#L110) so you can only add new things to the user, not remove things.

The alternative we figured was to:

1. fetch the team members
2. remove the user you don't want
3. update the team with that new list
4. done

But there is no current API that fetches these details per team.

### PR DESCRIPTION
**Pull request by Gby56**
Fixes #46

### CODE REVIEW

**Comment by tjarrettveracode** [comment]
@Gby56 , this is a pretty straightforward change, but could I ask for one change? In other methods in this area, we use `object().get(guid)` rather than the method name here (`get_by_id`). Can we change the method name so it's consistent with the other getters?

File: veracode_api_py/identity.py [hunk 1]
```diff
@@ -147,6 +147,10 @@ def get_all(self, all_for_org=False):
          request_params = {'page': 0} #initialize the page request
       return APIHelper()._rest_paged_request("api/authn/v2/teams","GET","teams",request_params)
 
+   def get_by_id(self, team_id):
```
Comment (tjarrettveracode) [inline veracode_api_py/identity.py:150]: Recommend changing method name just to `get()`

Comment (Gby56) [reply veracode_api_py/identity.py:150]: Sure, let me fix that

File: veracode_api_py/api.py [hunk 2]
```diff
@@ -241,6 +241,9 @@ def delete_user(self, user_guid: UUID):
 
     def get_teams(self, all_for_org=False):
         return Teams().get_all(all_for_org)
+    
+    def get_team_by_id(self, team_id):
+        return Teams().get_by_id(team_id)
```
Comment (tjarrettveracode) [inline veracode_api_py/api.py:247]: oops, looks like this now needs to be updated to call the new `Teams().get()` method

Comment (Gby56) [reply veracode_api_py/api.py:247]: Sorry about that, didn't really pay attention when I wrote that quickly

**Comment by tjarrettveracode** [comment]
Looks good, thanks!

**Pull request merged**